common/thorhelper/thorcommon.cpp

```diff
@@ -42,6 +42,8 @@
 
 #define ROWAGG_PERROWOVERHEAD (sizeof(AggregateRowBuilder))
 
+const cycle_t defaultFlushTimerInterval = nanosec_to_cycle(1000000000); // Cycles
```

Comment:
for ease of reading should comment that 1 second.

But doing this every second seem overkill.. The stats aren't collected that often..

Not sure what sensible value is, is 1 minute too long?

Reply:
Btw, if useful there's a oneSecInCycles constant defined in jdebug.hpp

Reply:
How frequently are the stats collected?  Could we use the same time period to update the stats? (Thanks for the info regardiong oneSecInCycles)